### ISSUE TITLE
no-whitespace-for-layout false positives

### ISSUE THREAD

**Issue by EvgenyOrekhov** [ember-template-lint/ember-template-lint#907]
```hbs
<div
  foo="bar"
  baz="qux"
>
  example
</div>

{{#foo-bar}}
  {{#baz-qux foo="bar"}}example{{/baz-qux}}
  {{#baz-qux foo="bar"}}example{{/baz-qux}}
{{/foo-bar}}

```

Actual result:

```text
  4:1  error  Excess whitespace detected.  no-whitespace-for-layout
  8:12  error  Excess whitespace detected.  no-whitespace-for-layout
  9:43  error  Excess whitespace detected.  no-whitespace-for-layout
```

Expected result: no warnings

**Comment by rwjblue** [comment]
Thanks for reporting!

**Comment by rwjblue** [comment]
@MelSumner - Mind taking a look?
